### DISCUSSION TITLE
What is the skill to be a good programmer?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#117480] [Programming Help] [answered]

**Comment by LiteBrite82** [comment, marked as answer]
Thanks for posting in the GitHub Community, @newprogrammers01 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
